packages/query-engine/source/middleware/errors/not-found-error.ts

```diff
@@ -0,0 +1,26 @@
+import { NextFunction, Response, Request } from "express";
+import { constants } from "@hypertool/common";
+
+const { httpStatuses } = constants;
+
+const NotFoundError = (
+    error: any,
+    request: Request,
+    response: Response,
+    next: NextFunction,
+): void => {
+    if (
+        error.code === "ER_BAD_DB_ERROR" ||
+        error.code === "ER_BAD_TABLE_ERROR" ||
+        error.code === "ER_NO_SUCH_TABLE"
+    ) {
+        response.status(httpStatuses.NOT_FOUND).json({
+            error: "Not Found",
```

Comment:
If this is an error code, `NOT_FOUND` is better. Also, wouldn't it be better if the user had more information than just "NOT_FOUND"? Let's retain the error codes. Create a mapping in constants.

ER_BAD_DB_ERROR -> BAD_DB_ERROR
ER_BAD_TABLE_ERROR -> BAD_TABLE_ERROR
NO_SUCH_TABLE -> NO_SUCH_TABLE_ERROR

and the other errors